public/app/features/integrated-alerting/components/Table/Pagination/Pagination.test.tsx

```diff
@@ -0,0 +1,311 @@
+import React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import { dataQa } from '@percona/platform-core';
+import { Pagination } from './Pagination';
+import { Messages } from './Pagination.messages';
+
+const simulateNextClick = (wrapper: ShallowWrapper, numberOfClicks: number) => {
+  for (let i = 0; i < numberOfClicks; i++) {
+    wrapper.find(dataQa('next-page-button')).simulate('click');
+  }
+};
+
+const simulatePreviousClick = (wrapper: ShallowWrapper, numberOfClicks: number) => {
+  for (let i = 0; i < numberOfClicks; i++) {
+    wrapper.find(dataQa('previous-page-button')).simulate('click');
+  }
+};
```

Comment:
nit: these look overkill, since this is a test and repeating the calls is clearer and understandable at fist glance

Reply:
In some tests we've 4 or 5 calls, is it really worth it copy/paste it?

Reply:
Well, if you think that's better, I'd use the loop explicitly instead of a function that readers will have to look up. If, on the other hand, you see that this function might be useful in other pieces of code, improving them, then you should move that function in our library, so that we know what's that for at first glance (but I don't think it's the case here) :)

Reply:
Just sticked with the loops 👍